test/linkedList.spec.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+const algolib = require('../index.js');
+
+describe('#linkedList', function () {
+    describe('#pushIntoEnd', function () {
+        it('should push element it the end of the list ', function () {
+            let linkedList = new algolib.LinkedList();
+            linkedList.pushIntoEnd(3);
+            linkedList.pushIntoEnd(7);
+            linkedList.pop().should.be.eql(7);
+        })
+    });
+    describe('#pushIntoStart', function () {
+        it('should push element in the start if the list', function () {
+            let linkedList = new algolib.LinkedList();
+            linkedList.pushIntoStart(1);
+            linkedList.pushIntoStart(2);
+            linkedList.pushIntoStart(4);
+            linkedList.pop().should.be.eql(1);
+            linkedList.pop().should.be.eql(2);
+            linkedList.pop().should.be.eql(4);
+
+        })
+    });
+    describe('#pop', function () {
+        it('should return last element', function () {
```

Comment:
how `#pushIntoEnd` and `#pop` test different from each other except description and number of elements?
do you really test different behavior?